frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -148,60 +157,67 @@ export function ObjectDetailFn({
   ]);
 
   const onKeyDown = (event: KeyboardEvent) => {
-    if (event.key === "ArrowLeft") {
+    if (event.key === "ArrowUp") {
       viewPreviousObjectDetail();
```

Comment:
Would be nice to call `preventDefault()` in order to prevent the grid from scrolling in the background